sycl/doc/extensions/supported/sycl_ext_oneapi_sub_group_mask.asciidoc

```diff
@@ -100,6 +113,38 @@ The mask is defined such that the least significant bit (LSB) corresponds to
 the work-item with id 0, and the most significant bit (MSB) corresponds to the
 work-item with the id `max_local_range()-1`.
 
+|===
+|Constructor|Description
```

Comment:
We should say somehow that all of these constructors and the assignment operator are available starting in revision 2 of this specification.

Reply:
Added corresponding note in 14c468290d540162908c8331758ab1db9cc57324